bin/pyenv-win-venv.ps1

```diff
@@ -44,53 +48,67 @@ function  main {
             if (test-path $python_version_file) {
                 $env_name = (Get-Content $python_version_file)
                 if ($env_name -And (test-path -PathType container "$app_env_dir\$env_name")) {
-                    &"$app_env_dir\$env_name\Scripts\Activate.ps1" 
+                    if ($subcommand1 -eq "ps1") {
+                        &"$app_env_dir\$env_name\Scripts\Activate.ps1" 
+                    }
+                    else {
+                        cmd /k "$app_env_dir\$env_name\Scripts\activate.bat"
+                    } 
                 }
             }
         }
     }
-    elseif ($subcommand1 -eq "activate") {
-        if (!$subcommand2) {
+    elseif ($subcommand2 -eq "activate") {
+        if (!$subcommand3) {
             HelpActivate
         }
-        elseif (test-path -PathType container "$app_env_dir\$subcommand2") {
-            &"$app_env_dir\$subcommand2\Scripts\Activate.ps1"
+        elseif (test-path -PathType container "$app_env_dir\$subcommand3") {
+            if ($subcommand1 -eq "ps1") {
+                $env:PYENV_VENV_ACTIVE = $subcommand3
+                &"$app_env_dir\$subcommand3\Scripts\Activate.ps1" 
+            }
+            else {
+                cmd /k "$app_env_dir\$subcommand3\Scripts\activate.bat"
+            }
             
         }
         else {
-            Write-Host "Env: $subcommand2 is not installed. Install using `"pyenv-win-venv install <python_version> $subcommand2"`"
+            Write-Host "Env: $subcommand3 is not installed. Install using `"pyenv-win-venv install <python_version> $subcommand3"`"
         }
     }
-    elseif ($subcommand1 -eq "deactivate") {
-        deactivate
+    elseif ($subcommand2 -eq "deactivate") {
+        DeactivatePyEnv
```

Comment:
It would have been cleaner to have the PR be branched of master and then fixed the small merge conflict that could have happened. Especially since changing deactivate between bat and ps1 isn't needed and there shouldn't be a big conflict anyhow.